Revision Tests/maryna_boieva.md

```diff
@@ -2,15 +2,15 @@
 
 **_1. Если бы я был умнее (по жизни) то я бы сделал вчера бэкап всей базы и только потом применял свой ПР к коду на проде. Жаль что я не умный как мой папа._**
 
-> If only I were more clever, I would have made a backup of the database yesterday and only after that apply my PR to production. I wish, i were as clever as my father.
+> If only I were more clever, I would have made a backup of the database yesterday and only after that apply my PR to production. I wish, I were as clever as my father.
 
 **_2. Если бы все студенты читали теорию и только потом писали перевод анализируя предложения, то прогресс бы был очевиден. Жаль что все хотят выучить английский брутфорсом, минуя ранние этапы обучения и анализа и отработки навыков до автоматизма. Я бы хотел чтобы вы не торопились._**
 
-> If all the students had read the theory and only after that wrote a translation analysing the sentences, the progress would be obvious. If only every one didn't tried to learn English brute force, escaping the early stages of analisis and skills training. I wish you didn't hurry up.
+> If all the students read the theory and only after that wrote a translation analysing the sentences, the progress would be obvious. If only every one didn't tried to learn English brute force, escaping the early stages of analisis and skills training. I wish you didn't hurry up.
```

Comment:
the translation (their translation)
If only everybod didn't try (we use infinitive after do,does, did)
with brute force
avoiding
analysis
I want you not to hurry up (complex object)